actionview/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Add the template digest to the cache key for collection caching.
```

Comment:
Yep, the `@virtual_path` isn't set in `AutomaticCollectionCacheTest` `test_explicit_render_call_with_options` or `test_caching_works_with_beginning_comment` because they're both rendering in the controller.

The other actions `render 'index'` and will have a `@virtual_path` of `collection_cache/index`.